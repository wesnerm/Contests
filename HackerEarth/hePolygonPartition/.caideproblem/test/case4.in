60
92 624
348 972
680 522
93 294
99 806
986 802
849 70
237 343
557 779
624 592
407 261
467 297
206 919
568 775
837 128
14 976
734 520
692 55
606 497
964 416
485 624
70 723
186 766
197 107
966 591
902 299
58 699
53 831
156 51
766 761
954 318
458 971
791 10
68 328
378 181
738 59
815 207
106 784
191 745
110 440
722 461
810 246
227 716
918 35
466 862
900 554
732 202
462 166
165 100
932 176
698 869
729 784
853 843
484 851
593 270
640 412
941 325
39 493
509 212
744 75