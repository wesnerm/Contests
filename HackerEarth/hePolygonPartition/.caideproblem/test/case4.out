15
2 19 56
4 16 29 44 6
4 28 49 7 53
4 5 24 50 51
4 4 15 30 13
4 34 33 52 38
4 58 48 20 32
4 40 18 25 45
4 22 8 60 46
4 1 59 57 54
4 27 35 31 2
4 39 36 26 9
4 23 11 37 14
4 43 12 42 17
6 21 55 47 41 3 10
